evm_loader/lib/src/account_storage.rs

```diff
@@ -159,13 +160,17 @@ impl<'rpc, T: Rpc> Rpc for EmulatorAccountStorage<'rpc, T> {
         &self,
         pubkeys: &[Pubkey],
     ) -> ClientResult<Vec<Option<Account>>> {
-        // TODO: Optimize this!!!
-        let mut result = vec![];
-        for key in pubkeys {
-            let account = self.get_account(key).await?.value;
-            result.push(account);
+        let results =
+            futures::future::try_join_all(pubkeys.iter().map(|key| self.get_account(key))).await;
```

Comment:
You try to request accounts in parallel. This leads to many requests to the Solana node. It's better to use the already implemented method Rpc::get_multiple_accounts https://github.com/neonlabsorg/neon-evm/blob/7e70f95cb18afb6148db0fc17ee19c4c2f42cbb9/evm_loader/lib/src/rpc/mod.rs#L31 and write the function like `_get_account_from_rpc` (one of the implementation is at https://github.com/neonlabsorg/neon-evm/blob/699c6aec58a60e589e52796e65cafaa370a8b64d/evm_loader/lib/src/rpc/validator_client.rs#L72).

Reply:
done